packages/salesforce-adapter/src/metadata_deploy.ts

```diff
@@ -320,6 +325,13 @@ const processDeployResponse = (
       severity: 'Error' as SeverityLevel,
     }))
 
+  if (failedComponentErrors.some((error) => error.elemID === undefined)) {
+    log.debug(
+      'Some deploy messages could not be mapped to an ElemID. typeAndNameToElemId=%s',
```

Comment:
Is this potentially a large map?
Also should this be %o
```suggestion
      'Some deploy messages could not be mapped to an ElemID. typeAndNameToElemId=%o',
```